libnmstate/nm/device.py

```diff
@@ -20,13 +20,28 @@
 from . import nmclient
 
 
-def activate(dev, connection=None):
+def activate(dev=None, connection_id=None):
+    """
+    Activate the given device or remote connection profile.
+    For the remote connection profile, a fetcher function is provided which
```

Comment:
The docstring needs to be adjusted for the change from fetch function to connection_id

Reply:
Done